### ISSUE TITLE
Too many surfaces error (texinfo maximum)

### ISSUE THREAD

**Issue by Ypsckud** [yquake2/yquake2#1053]
There seems to be a 8192 texinfo maximum in the engine. Can it be increased?

**Comment by 0lvin** [comment]
Do you have map example with bigger amount of texinfo? It could require changes in all renders to support bigger maps.

**Comment by Ypsckud** [comment]
Yes, it's the map I am currrenlty making. I think that 8192 is a pretty small number considering that there are a lot of textures to be used with unique offsets for modern mapping (like trim textures). I managed to run the map on re-release version of Q2, but not on Yamagi (so the map itself is OK)

**Comment by 0lvin** [comment]
You could try branch with partial [ReRelease](https://github.com/0lvin/yquake2/tree/rerelease-combined) support, but I don't know when I finish code changes or when it could be merged, if merged/finished at all.

**Comment by Ypsckud** [comment]
Will do, thanks for answering!

**Comment by Yamagi** [comment]
I'm very hesitant to change these kind of things in YQ2 because these are that kind of changes which seem to work well until someone finds that one case which doesn't. However this is a good argument for doing Q2 Remastered support in a separate branch with relaxed backward compatibility considerations.
